src/util/test-data.js

```diff
@@ -212,6 +213,33 @@ export const createReview = (id, attributes = {}, includes = {}) => {
   };
 };
 
+/**
+ * Creates an array of time slot objects.
+ *
+ * @param {Date} startDate date when the time slots start
+ * @param {Number} numberOfDays number of time slots to create
+ *
+ * @return {Array} array of time slots
+ */
+export const createTimeSlots = (startDate, numberOfDays) => {
+  const startTime = moment(startDate).startOf('day');
+
+  return Array.from({ length: numberOfDays }, (v, i) => i).map(i => {
+    return {
+      id: new UUID(i),
+      type: 'timeSlot',
+      attributes: {
+        start: moment(startTime)
```

Comment:
This needs to use UTC date - otherwise, customizers get errors if they run this (ie. their laptop) in different timezones than UTC.

Reply:
Hmm. This is only called from styleguide - so, it's not needed for test verification. I'm not sure if this will cause problems or not. (I was assuming that startDate should be something like `dateFromLocalToAPI(startDate)`, but moment's `startOf` function doesn't know about UTC start of day moment anyway...)